src/Ions.cc

```diff
@@ -967,7 +970,8 @@ void Ions::readRestartPositions(HDFrestart& h5_file)
         (*MPIdata::sout) << "Read ionic positions from hdf5 file" << std::endl;
 
     std::vector<double> data;
-    h5_file.readAtomicPositions(data);
+    std::string datasetname("/Ionic_positions");
```

Comment:
@jeanlucf22 Is there a reason why we need the '/' for some of the data set names and not others? For example, the GidList does not have it.

Reply:
I am not sure. I did not change that. But indeed it may not be needed